src/main/webapp/script.js

```diff
@@ -86,13 +87,37 @@ function clearElm(elementID) {
 
 function userResult() {
     clearElm(QUIZ_ELEMENT_ID);
-    const quiz = document.getElementById(QUIZ_ELEMENT_ID);
     quiz.appendChild(createParagraphElement("Based On Your Results, You Should visit..."));
```

Comment:
if you plan on appending the "Based On ..." I think you may need to keep the document.getElementByID?